### ISSUE TITLE
 The intent the app is registering updates for is not found

### ISSUE THREAD

**Issue by zehrasubas** [actions-on-google/actionssdk-updates-nodejs#3]
Hello,

When I click Register daily updates suggestion chip and selecting category it gives error as: The intent the app is registering updates for is not found.

It seems it needs to call tell.tip intent. I've checked from the code and tell.tip intent is described correctly. Can you please guide on why this issue is happening.

**Comment by asjgi** [comment]
How could you solve this problem?

**Comment by zehrasubas** [comment]
Hi,
You need to define Content title for update from AoG console-> Actions-> Select related intent -> Define content title under User Engagement. That should solve it.

**Comment by asjgi** [comment]
Hello, Thanks a lot reply to me.
Can I ask you more question?
I don`t understand exactly what you said.
when I open that site. (AoG console-> Actions-> Select related intent -> Define content title)
It is not possible to make another action.
So I tried to make an action at the dialogflow directly.
I don`t know it is right to make it.
I made like that flow the picture.
Cloud you check for me?

![image](https://user-images.githubusercontent.com/6354767/48538560-2419bc80-e8f8-11e8-9598-34a4e88766bc.png)
![image](https://user-images.githubusercontent.com/6354767/48538444-d7ce7c80-e8f7-11e8-9bb2-5ae695a8c082.png)
![image](https://user-images.githubusercontent.com/6354767/48538681-6fcc6600-e8f8-11e8-9141-69470afd41c1.png)
![image](https://user-images.githubusercontent.com/6354767/48538899-0567f580-e8f9-11e8-8264-554dba2132f2.png)

And I gonna try to ask you last of a question.
Is it possible to directly notification?
Actually, I want to notify directly when I received the call from the other intent function.
Have you tried to make it like this(Direct notification)?
I want to make like that flow picture.
![image](https://user-images.githubusercontent.com/6354767/48540557-c38d7e00-e8fd-11e8-863b-a9813c68694d.png)

from David.ahn

**Comment by zehrasubas** [comment]
You shouldn't open Dialogflow stay at AoG console to define content title. After clicking Actions on the console, click one of the intents (eg. Default Welcome Intent). At the bottom of the page you should see User Engagement tab, from here you need to enable updates and add content title and save.

**Comment by asjgi** [comment]
I think I can`t.
When I push the add action button It guides me Dialogflow page.
![image](https://user-images.githubusercontent.com/6354767/48599748-67356780-e9ac-11e8-9dab-e24583576ae0.png)

**Comment by Canain** [comment]
@asjgi If your project is using Dialogflow, you should use the [Dialogflow Updates Sample](https://github.com/actions-on-google/dialogflow-updates-nodejs) instead as this one is for Actions SDK.

**Comment by asjgi** [comment]
@Canain Thank you for a reply to me, I already check that source code but I don`t understand exactly, How can I use at my project.

First, I made the one of Action (before I can not make it) by the dialogflow of integration setting
(Actually, I set the 2 languages(Korea, English), but I did not write down anything second language(Korea) intent of Training phrases )
https://productforums.google.com/forum/#!topic/dialogflow/7ptMg57TVBI


When I made a new action by the dialogflow integration setting option.
I can see the error notification like the (This intent is consist of invalid parameter ~ blabla)
So, I could not make a new action. 
But Now I can make a new Action at the AOG consol, I`m gonna make a new step.
Thank you, guys. If I have a problem, I gonna come back.
